src/models/parallelCoordinatesChart.js

```diff
@@ -0,0 +1,331 @@
+		                if (d.currentPosition !== d.originalPosition)
+		                    isSorted = true;
+		            });
+		            if (isSorted)
+		                resetSortingButton.style("visibility", "visible");
+		            else
+		                resetSortingButton.style("visibility", "hidden");
+
+		            dispatch.dimensionsOrder(dimensionNames);
+		        });
+
+				// Update chart from a state object passed to event handler
+                dispatch.on('changeState', function (e) {
+
+                    if (typeof e.disabled !== 'undefined') {
+                        dimensionNames.forEach(function (series, i) {
+                            series.disabled = e.disabled[i];
+                        });
+
+                        state.disabled = e.disabled;
+                    }
+
+                    chart.update();
+                });
+            });
+
+            renderWatch.renderEnd('parraleleCoordinateChart immediate');
+            return chart;
+        }
+
+		//============================================================
+        // Event Handling/Dispatching (out of chart's scope)
+        //------------------------------------------------------------
+
+        parallelCoordinates.dispatch.on('elementMouseover.tooltip', function (evt) {
+            evt['series'] = {
+                key: evt.label,
+                color: evt.color
+            };
+            tooltip.data(evt).hidden(false);
+        });
+
+        parallelCoordinates.dispatch.on('elementMouseout.tooltip', function(evt) {
+            tooltip.hidden(true)
+        });
+
+        parallelCoordinates.dispatch.on('elementMousemove.tooltip', function () {
+            tooltip.position({ top: d3.event.pageY, left: d3.event.pageX })();
+        });
+		 //============================================================
+        // Expose Public Variables
+        //------------------------------------------------------------
+		
+		// expose chart's sub-components
+        chart.dispatch = dispatch;
+        chart.parallelCoordinates = parallelCoordinates;
+        chart.legend = legend;
+        chart.tooltip = tooltip;
+
+        chart.options = nv.utils.optionsFunc.bind(chart);
+
+        chart._options = Object.create({}, {
+            // simple options, just get/set the necessary values
+            width: { get: function () { return width; }, set: function (_) { width = _; } },
+            height: { get: function () { return height; }, set: function (_) { height = _; } },
+            showLegend: { get: function () { return showLegend; }, set: function (_) { showLegend = _; } },
+            showControls: { get: function () { return showControls; }, set: function (_) { showControls = _; } },
+            defaultState: { get: function () { return defaultState; }, set: function (_) { defaultState = _; } },
+            dimensionData: { get: function () { return dimensionData; }, set: function (_) { dimensionData = _; } },
```

Comment:
You removed dimentionFormats... you should keep it but add a "this is deprecated" message like we did with several other options... also include a comment about what version it was deprecated after so we know when we can remove it completely.